packages/list-reusable-blocks/src/utils/file.js

```diff
@@ -6,11 +6,22 @@
  * @param {string} contentType File mime type.
  */
 export function download( fileName, content, contentType ) {
-	const a = document.createElement( 'a' );
 	const file = new window.Blob( [ content ], { type: contentType } );
-	a.href = URL.createObjectURL( file );
-	a.download = fileName;
-	a.click();
+
+	// IE11 can't use the click to download technique
+	// we use a specific IE11 technique instead.
+	if ( window.navigator.msSaveOrOpenBlob ) {
+		window.navigator.msSaveOrOpenBlob( file, fileName );
+	} else {
+		const a = document.createElement( 'a' );
+		a.href = URL.createObjectURL( file );
```

Comment:
Not sure how I feel about this cache, seems like a memory waste at first sight. Do you know for which case it was introduced originally?

Reply:
I think it was for the drag&drop and copy&past file uploads, we use a blob URL temporarily and then we want to from the blob URL get the original file object that contains metadata like the file name.

I'm not sure about the memory waste, the browser needs to keep blobs in memory anyway, and I expect that the browser reuses the file. Probably the browser uses the bytes of the file as the place where the blob points to (to avoid the need to copy them). So perhaps this bytes would still be needed, but I'm not sure about what happens in this cases.